app/containers/ItuensContainer/apiUtils.ts

```diff
@@ -0,0 +1,18 @@
+import { create } from 'apisauce';
+
+const api = create({ baseURL: 'https://itunes.apple.com' });
```

Comment:
apiUtils will not be container specific, this should be a separate file in the services folder